Fix #60: improve run-example OpenAMASE resolution

manthonyaiello opened the pull request:
This commit refactors the development-directory resolution for LmcpGen and OpenAMASE. The functions are moved into `paths.py` so they can be shared across scripts; the associated command-line arguments are also moved to `paths.py` and encapsulated in new functions.

`run_lmcpgen.py` is updated to use these new functions correctly.

`run_example.py` is also updated to use the resolution of OpenAMASE. This makes the behavior consistent between run-example and run-lmcpgen.

Resolves issue #60.

derekkingston commented:
Changing the CMASI MDM has ramifications on backwards compatibility. It's highly preferred that MDMs remain constant. Here you are adding a discretization value to PointSearchTask which should instead use a parameter via the configuration file. An example of a task that uses a configuration parameter is the LineSearchTask which allows the search to be denoted as "single direction". Please change the PointSearchTask to use a discretization parameter from the configuration file.

manthonyaiello committed:
Fix #60: improve run-example OpenAMASE resolution

This commit refactors the development-directory resolution for LmcpGen
and OpenAMASE. The functions are moved into `paths.py` so they can be
shared across scripts; the associated command-line arguments are also
moved to `paths.py` and encapsulated in new functions.

`run_lmcpgen.py` is updated to use these new functions correctly.

`run_example.py` is also updated to use the resolution of OpenAMASE.
This makes the behavior consistent between run-example and run-lmcpgen.

Resolves issue #60.

manthonyaiello commented:
Testing note: you need to `rm -rf .vpython` because the `uxas` python module changed and needs to be reinstalled.
